### ISSUE TITLE
Freshly launch of IMAGE containers causes failure to return renderings due to preprocessor errors

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-server#848]
tldr: If you relaunch all the IMAGE containers on unicorn, and then try and get renderings for the first photo on the IMAGE homepage, it fails repeatedly. If you then try an indeterminate number of times to get renderings of the second photo (people clasping hands), you'll get valid renderings returned. After this, the first photo works again.

test version of extension 0.4.71
make sure in options you're using unicorn for server

log into unicorn
cd `/var/docker/image`
`./restoreunstable`
wait for all containers to restart
Go to IMAGE homepage in browser
Right click photo of person sitting at computer
Get IMAGE rendering
NOTE: No renderings returned
NOTE: Logs on unicorn filled with errors like:

```
object-depth-calculator-1         | 2024-07-09T20:19:45.328336406Z [2024-07-09 20:19:45 +0000] [7] [DEBUG] POST /preprocessor                            
object-depth-calculator-1         | 2024-07-09T20:19:45.382524150Z [2024-07-09 20:19:45,382] ERROR in object-depth-calculator: '' does not match '^[A-Za-z]{2,6}(\\.((?!-)[A-Za-z0-9-]+(?<!-))
)+$'                                                                                                                                                                                          
object-depth-calculator-1         | 2024-07-09T20:19:45.382542946Z                                                                                                                            
object-depth-calculator-1         | 2024-07-09T20:19:45.382549307Z Failed validating 'pattern' in schema['properties']['preprocessors']['propertyNames']:                                     
object-depth-calculator-1         | 2024-07-09T20:19:45.382554988Z     {'pattern': '^[A-Za-z]{2,6}(\\.((?!-)[A-Za-z0-9-]+(?<!-)))+$'}                                                         
object-depth-calculator-1         | 2024-07-09T20:19:45.382560869Z                                                                                                                            
object-depth-calculator-1         | 2024-07-09T20:19:45.382566219Z On instance['preprocessors']:                                                                                              
object-depth-calculator-1         | 2024-07-09T20:19:45.382571679Z     ''                                                                                                                     
ocr-clouds-preprocessor-1         | 2024-07-09T20:19:45.392719159Z ERROR:root:'' does not match '^[A-Za-z]{2,6}(\\.((?!-)[A-Za-z0-9-]+(?<!-)))+$'                                             
ocr-clouds-preprocessor-1         | 2024-07-09T20:19:45.392737874Z                                                                                                                            
ocr-clouds-preprocessor-1         | 2024-07-09T20:19:45.392744246Z Failed validating 'pattern' in schema['properties']['preprocessors']['propertyNames']:                                     
ocr-clouds-preprocessor-1         | 2024-07-09T20:19:45.392750037Z     {'pattern': '^[A-Za-z]{2,6}(\\.((?!-)[A-Za-z0-9-]+(?<!-)))+$'}                                                         
ocr-clouds-preprocessor-1         | 2024-07-09T20:19:45.392755878Z                                                                                                                            
ocr-clouds-preprocessor-1         | 2024-07-09T20:19:45.392761228Z On instance['preprocessors']:                                                                                              
ocr-clouds-preprocessor-1         | 2024-07-09T20:19:45.392766838Z     ''                                                                                                                     
object-grouping-1                 | 2024-07-09T20:19:45.399235497Z ERROR:root:'' does not match '^[A-Za-z]{2,6}(\\.((?!-)[A-Za-z0-9-]+(?<!-)))+$'        
object-grouping-1                 | 2024-07-09T20:19:45.399254904Z                                                                                                                            
object-grouping-1                 | 2024-07-09T20:19:45.399260785Z Failed validating 'pattern' in schema['properties']['preprocessors']['propertyNames']:                                     
object-grouping-1                 | 2024-07-09T20:19:45.399266565Z     {'pattern': '^[A-Za-z]{2,6}(\\.((?!-)[A-Za-z0-9-]+(?<!-)))+$'}                                                         
object-grouping-1                 | 2024-07-09T20:19:45.399272466Z                                                                                                                            
object-grouping-1                 | 2024-07-09T20:19:45.399278578Z On instance['preprocessors']:                                                                                              
object-grouping-1                 | 2024-07-09T20:19:45.399284228Z     ''                                                                                                                     
orchestrator-1                    | 2024-07-09T20:19:45.399877358Z Error occured on fetch from http://object-grouping:5000/preprocessor                                                       
orchestrator-1                    | 2024-07-09T20:19:45.399898678Z Invalid Preprocessor JSON format                                                                                           
orchestrator-1                    | 2024-07-09T20:19:45.399914788Z Error occured on fetch from http://ocr-clouds-preprocessor:5000/preprocessor                                               
orchestrator-1                    | 2024-07-09T20:19:45.399930537Z Invalid Request JSON format                                                                                                
orchestrator-1                    | 2024-07-09T20:19:45.399944724Z Error occured on fetch from http://object-depth-calculator:5000/preprocessor                                               
orchestrator-1                    | 2024-07-09T20:19:45.399974480Z Now on priority group 5                                                             
```

EXPECT: Renderings returned

repeat trying to get renderings for that photo
NOTE: Never seems to work, continues to error out on server and not return any renderings

right click photo of two people clasping hands on IMAGE homepage
get IMAGE renderings
NOTE: May return renderings. If not, try a few times, and it will.

once you get a valid rendering from the second photo on the homepage:
right click photo of person sitting at computer
get IMAGE rendering
NOTE: Valid renderings returned

Once it starts working, it seems to keep working. As if something is not being initialized, but it eventually gets tickled in the right way, and then everything starts working.

This is an issue since if unicorn reboots and we don't notice, it could be in a bad state for a while. Logging as an end user production issue since we don't want people hitting the server and having problems.
